docs/configuring-entities.md

```diff
@@ -0,0 +1,197 @@
+# Configuring catalog entities
+
+Catalog entities are configurable through YAML definition files which can be found [here][1]. Documentation on configuring entities without using any plugins can be found in the [upstream documentation][2]. The sections below describe how to configure installed plugins for catalog entities.
+
+When creating new entities this [ADR][16] should be followed.
+
+## TechDocs plugin
+
+TechDocs plugin handles the building and the publishing of your documentation. For this plugin an annotation is not enough to configure it. Firstly [create the necessary file structure](#mkdocs) for your documentation. A [GitHub workflow][10] is required to on the repository that hosts the documentation source files. For a guide on how to [setup this workflow](#using-techdocs-workflow-template) check the subsection below. After the workflow is setup an url reference to your documentation is required in the catalog entity definition file. This url will not be used a reference for the documentation builder but as a reference to the source files in the catalog so that users can find the source files. See the example for an annotation below.
```

Comment:
Definitely makes more sense. I also split the techdocs configuration into 3 steps instead of 2 since it feels like the first step are actually two steps.